packages/cli/src/commands/ts.ts

```diff
@@ -126,7 +129,7 @@ export default class TS extends Command {
                   if (compareVersions.compare(ctx.latestELRdBuild, ctx.currentTSVersion, '>')) {
                     ctx.tsVersion = ctx.latestELRdBuild;
                     parent.title = `Rehearsing with typescript@${ctx.tsVersion}`;
-                    reporter.tscVersion = ctx.tsVersion;
+                    //reporter.tscVersion = ctx.tsVersion;
```

Comment:
we will want to be sure we are exposing the version of typescript compiler we are rehearsing against